CustomerService/Service/CustomerDataService.cs

```diff
@@ -23,23 +24,65 @@ public CustomerDataService(ICustomerRepository _customerRepository, IDomainEvent
         }
```

Comment:
This class should be called CustomerService

Reply:
Done